_posts/2019-01-08-linguistics-minute-rolf-the-ganger.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: "Linguistics Minute: Rolf the Ganger"
+date: "2019-01-08 12:00:00 -0500"
+---
+_This was originally published in my work Slack on 2016-10-06, but I've since
+been convinced that more people than just those few who can even read it there
+might be interested. I've edited it for clarity and correctness. I'm going to
+migrate all my Linguistcs Minutes over to my blog and probably post future ones
+here._
+
+In the north of France, there's a region called Normandy. So called because it
```

Comment:
#grammar "So called..." opens a dependent clause. Join to the prior clause with a comma or promote to a complete sentence, and ideally avoid close repetition of "called".